docs/recipes/travis-build-stages.md

```diff
@@ -1 +1,114 @@
 # Using semantic-release with [Travis CI build stages](https://docs.travis-ci.com/user/build-stages)
+
+## Environment variables
+
+The [Authentication](../usage/ci-configuration.md#authentication) environment variables can be configured in [Travis Repository Settings](https://docs.travis-ci.com/user/environment-variables/#defining-variables-in-repository-Settings) or with the [travis env set CLI](https://github.com/travis-ci/travis.rb#env).
+
+Alternatively, the default `NPM_TOKEN` and `GH_TOKEN` can be easily [setup with semantic-release-cli](../usage/ci-configuration.md#automatic-setup-with-semantic-release-cli).
+
+## Multiple Node jobs configuration
+
+### `.travis.yml` configuration for multiple Node jobs
+
+This example is a minimal configuration for **semantic-release** with a build running Node 4, 6 and 8 on Linux and OSX. See [Travis - Customizing the Build](https://docs.travis-ci.com/user/customizing-the-build) for additional configuration options.
+
+This example creates a `release` [build stage](https://docs.travis-ci.com/user/build-stages) that [runs `semantic-release` only after all test jobs are successful](../usage/ci-configuration.md#run-semantic-release-only-after-all-tests-succeeded).
+
+It's recommended to run the `semantic-release` command in the [Travis `deploy` step](https://docs.travis-ci.com/user/customizing-the-build/#The-Build-Lifecycle) so if an error occurs the build will fail and Travis will send a notification.
+
+**Note**: It's not recommended to run the `semantic-release` command in the Travis `script` step as each script in this step will be executed regardless of the outcome of the previous one. See [travis-ci/travis-ci#1066](https://github.com/travis-ci/travis-ci/issues/1066).
+
+**Advanced configuration**: Running the tests in the `script` step of the `release` stage is not necessary as the previous stage(s) already ran them. To increase speed, the `script` step of the `release` stage can be overwritten to skip the tests. Note that other commands such as build or compilation might still be required.
+
+```yaml
+language: node_js
+
+node_js:
+  - 8
+  - 6
+  - 4
+
+os:
+  - linux
+  - osx
+
+# Run the default test stage first, then the release one only when all the test jobs are successful
+stages:
+  - test
+  - release
```

Comment:
I'm not sure we need this, I think that is the default?

Reply:
Indeed it's the default and we don't need it. I added it for more clarity.
But we can remove it.
Do you think it's more confusing to keep it or to remove it?

Reply:
I removed it